### PR TITLE
Almighty judgment

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by DeathlyPlays** [comment]
>_>

**Comment by ghost** [comment]
I'm using it for travis because I can't check it locally on an ipad

**Comment by PictureIfYouWill** [comment]
.>

**Comment by DeathlyPlays** [comment]
Ah you can use travis-ci.org

**Comment by DeathlyPlays** [comment]
It actually shows you that on that branch there's travis probs

**Comment by ghost** [comment]
<s>now you tell me</s>

**Comment by ghost** [comment]
At least I deleted the conflicts

**Review by DeathlyPlays — COMMENTED** [review]
Meh, I'll test it

**Comment by DeathlyPlays** [comment]
Looks fine to me lol

**Comment by ghost** [comment]
I used revelation dance's code instead of synchronoise to determine the type of amazing plate

**Comment by DeathlyPlays** [comment]
Discord @PictureIfYouWill

File: mods/essb/items.js [hunk 1]
```diff
@@ -176,8 +176,40 @@ exports.BattleItems = {
...
+		onTakeItem: function (item, pokemon, source) {
+			if ((source && source.baseTemplate.num === 572) || pokemon.baseTemplate.num === 572) {
+				return false;
+			}
+			return true;
+		},
+		onResidualOrder: 5,
+		onResidualSubOrder: 2,
+		onResidual: function (pokemon, target, source) {
+			if (target.hasType(source.getTypes())) {
```
Comment (DeathlyPlays) [inline mods/essb/items.js]: getTypes isn't defined

**Comment by ghost** [comment]
Luckily I have insurance

File: mods/essb/items.js (hunk 1 above)
Comment (ghost) [reply mods/essb/items.js]: Deleted

**Comment by ghost** [comment]
@DeathlyPlays ready for testing

**Comment by ghost** [comment]
Last thing i edited so everything should work

**Pull request merged**